.travis.yml

```diff
@@ -9,6 +9,10 @@ python:
   - 3.6
   - 3.7
   - 3.8
+  - 3.9 
```

Comment:
You can undo changes in this file

Reply:
I undo this changes as per you requested !